growth/too/tasks/growthdb_cgi.py

```diff
@@ -19,14 +21,105 @@
 """
 
 
+def prepare_candidates_for_object_table(sources_all):
```

Comment:
This shouldn't be a function at all. This all needs to go into the Jinja template.

Reply:
Function removed.